applications/GeoMechanicsApplication/tests/dirichlet_u_constant/ProjectParameters_stage3.json

```
@@ -0,0 +1,149 @@
+{
+    "problem_data": {
+        "problem_name":         "dirichlet_u_constant",
+        "start_time":           2.0,
+        "end_time":             3.0,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "U_Pw",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "dirichlet_u_constant"
+        },
```

Also in the third stage, there is no need to import the model once again:

```suggestion
        "model_import_settings":              {
            "input_type":       "use_input_model_part"
        },
```

---

done